packages/CollabCodeEdit-Server.package/CCEDeployAgent.class/class/agentPort.st

```diff
@@ -1,3 +1,5 @@
 constants
 agentPort
-	^ (self envVar: 'CCE_AGENT_PORT') ifNil: [ 9999 ]
\ No newline at end of file
+	^ (self envVar: 'CCE_AGENT_PORT')
```

Comment:
Is `envVar:` in standart Smalltalk? If not, that name is akward.

Reply:
No it isn't (see `CCEDeployAgent>>envVar:`). Why is it that you think the name is awkward?

Reply:
In squeak names should be read like english sentences, not like abbreviations.